src/test/java/edu/cornell/kfs/coa/fixture/AccountFixture.java

```diff
@@ -0,0 +1,64 @@
+package edu.cornell.kfs.coa.fixture;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.kuali.kfs.coa.businessobject.Account;
+import org.kuali.kfs.coa.businessobject.IndirectCostRecoveryAccount;
+import org.kuali.rice.krad.util.ObjectUtils;
+
+public enum AccountFixture {
+	ACCOUNT_1111111_2222222_98_2(
+			IndirectCostRecoveryAccountFixture.ICR_1111111_98_PERCENT_ACTIVE
+					.getIndirectCostRecoveryAccountChange(),
+			IndirectCostRecoveryAccountFixture.ICR_2222222_2_PERCENT_ACTIVE
+					.getIndirectCostRecoveryAccountChange(), null),
+
+	ACCOUNT_1111111_98_INACTIVE_98_INACTIVE_3333333_100_PERCENT_ACTIVE(
+			IndirectCostRecoveryAccountFixture.ICR_1111111_98_PERCENT_INACTIVE
+					.getIndirectCostRecoveryAccountChange(),
+			IndirectCostRecoveryAccountFixture.ICR_1111111_98_PERCENT_INACTIVE
+					.getIndirectCostRecoveryAccountChange(),
+			IndirectCostRecoveryAccountFixture.ICR_3333333_100_PERCENT_ACTIVE
+					.getIndirectCostRecoveryAccountChange()),
+
+	ACCOUNT_3333333_100(
+			IndirectCostRecoveryAccountFixture.ICR_3333333_100_PERCENT_ACTIVE
+					.getIndirectCostRecoveryAccountChange(), null, null),
+
+	ACCOUNT_3333333_100_INACTIVATE(
+			IndirectCostRecoveryAccountFixture.ICR_3333333_100_PERCENT_ACTIVE
+					.getIndirectCostRecoveryAccountChange(),
+			IndirectCostRecoveryAccountFixture.ICR_1111111_98_PERCENT_INACTIVE
+					.getIndirectCostRecoveryAccountChange(),
+			IndirectCostRecoveryAccountFixture.ICR_2222222_2_PERCENT_INACTIVE
+					.getIndirectCostRecoveryAccountChange());
+
+	public IndirectCostRecoveryAccount icrAccount1;
+	public IndirectCostRecoveryAccount icrAccount2;
+	public IndirectCostRecoveryAccount icrAccount3;
+
+	private AccountFixture(IndirectCostRecoveryAccount icrAccount1,
+			IndirectCostRecoveryAccount icrAccount2,
+			IndirectCostRecoveryAccount icrAccount3) {
```

Comment:
Another fixture that could use an ICR Account array and a var-args constructor parameter.